shell/components/ResourceDetail/Masthead.vue

```diff
@@ -370,7 +374,7 @@ export default {
     },
 
     hideNamespaceLocation() {
-      return this.$store.getters['currentProduct'].hideNamespaceLocation;
+      return this.$store.getters['currentProduct'].hideNamespaceLocation || !this.value?.localCluster;
```

Comment:
`this.$rootGetters['management/byId'](MANAGEMENT.CLUSTER, LOCAL_CLUSTER)` can be used here directly

Reply:
i realise now this was the wrong suggestion! i should have commented here about using resource specific info instead of recommending the above line. Apologies